libnetwork/driver_plugin.py

```diff
@@ -306,3 +531,63 @@ def remove_veth(name):
         netns.remove_veth(name)
     except CalledProcessError:
         app.logger.warn("Failed to delete veth %s", name)
+
+
+def get_network_data(network_id):
+    """
+    Return the network data, and raise an exception if the network does not
```

Comment:
Might be nicer to say what "network data" actually is - i.e. the JSON that's passed in on the CreateNetwork request.